src/styles/_AddRecipe.scss

```diff
@@ -0,0 +1,19 @@
+.header1 {
+    text-align: left;
+    margin-top: 50px;
+}
+
+.header2 {
+    margin-top: 50px;
+}
+
+.btn1 {
+    height: 40px;
+    position: relative;
+    top: 22px;
+}
+
+.btn2 {
+    position: relative;
+    top: 50px;
+}
```

Comment:
What is the purpose of having btn1 and btn2 styling? It might be better if these had more descriptive names. Additionally there is global button styling that you should be using.

Reply:
The buttons are still not using the global button styling.